client/src/components/breadcrumbs.jsx

```diff
@@ -0,0 +1,56 @@
+import { useState, useEffect } from 'react'
+import PropTypes from 'prop-types'
+import './Breadcrumbs.css'
+
+// internal logic for breadcrumbs:
+// "/"   -->  ["home"]
+// "/one" --> ["home", "one"]
+// "/one/one_one" --> ["home", "one", "one_one"] --> home / one / one_one
+//
+// hint:
+// ["home", "one", "one_one"].join("/")
+
+const Breadcrumbs = ({ currentPath, fetchFiles }) => {
+  const [breadcrumbs, setBreadcrumbs] = useState([])
+
+  useEffect(() => {
+    const pathnames = currentPath === '/' ? [``] : currentPath.split('/').slice(1)

```

Comment:
if path === "/"  then is need to be displayed as "home". 
please see comment above at the line # 6